Documentation/coding-guidelines/clr-jit-coding-conventions.md

```diff
@@ -16,6 +16,8 @@ Note that these conventions are different from the CLR C++ Coding Conventions, d
 
 > Note: the JIT currently doesn't follow some of these conventions very widely. The non-conformant code should be updated, eventually.
 
+> Node: we now use jit-format to format our code. All changes it makes supersedes the conventions in this doc. Please see the [https://github.com/dotnet/jitutils/blob/master/doc/getstarted.md](jit-format documentation) for instructions on running jit-format.
```

Comment:
Also, you may want to use https://github.com/dotnet/jitutils/blob/master/doc/getstarted.md#formatting-jit-source for the URL instead of just https://github.com/dotnet/jitutils/blob/master/doc/getstarted.md